### PR TITLE
Survey reminder

### LINKED ISSUE #668: [FEAT] Allow user to navigate to HealthLog without completing survey
# ⚠️ IMPORTANT: Please fill out this template to give us as much information as possible to consider/implement the feature.

### Prerequisites

- [x] check this box if you have completed the following:
    - Reviewed the [contributing guidelines](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md) and [support](https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md) files
    - Reviewed the README file for the repository you are working in
    - Searched for relevant instructions [on our Discord server](https://discord.gg/pPERUuv)
    - Searched the issues of the repository you are working in to make sure one was not already filed

### Summary
Now when the user logs in for the first time is promoted to do the survey. If the user doesn't want to do that, the user can't access `HealthLog` until completing the survey.
we want to promote the user to take the survey upon login but allow to go to `healthLog` without taking the survey 

### Motivation
Give the user the choice when to take the survey 

### Possible Alternatives
remove this function and redirect to `healthLog` upon login 

### Additional Context
We can make a way later to remind the user to take the survey like the idea in #662 or any other way

### PR DESCRIPTION
**Pull request by PhurbaSherpa**
# ⚠️ __IMPORTANT: Please do not create a Pull Request without creating an issue first.__

All changes need to be discusssed before proceeding. Failure to do so may result in the pull request being rejected.

Please be sure to review our [Code of Conduct](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CODE_OF_CONDUCT.md), [Contributing Guidelines](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md), and [Support documentation](https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md) before submitting a pull request.

-----

## Please include the issue number the pull request fixes by replacing YOUR-ISSUE-HERE in the text below.

Fixes #668 

## Summary

Gives a user a reminder every time they log in to take their survey if they haven't taken one that day.

If a user deletes their data they'll be reminded to take a survey upon going back to the health Log.
## Details

Before a user was forced to take a survey if they haven't that day before being able to navigate the website.

## Test Plan (required)

reminder

## Final Checklist

- [x] For CoronaTracker, did you bump the version in `package.json`?
    - Update the second decimal for a major change
    - Update the third decimal for a minor change
    - Numbers can go past 9, e.g. `1.0.9` => `1.0.10`
    - For more info, read about [Semantic Versioning](https://semver.org/)
- [x] Did you add any new tests as necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits? _(can be done at merge)_
- [x] Did you use `yarn` not `npm`? _(important!)_
- [x] Did you use Material-UI wherever possible?
- [x] Did you run `yarn lint` on the code?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

File: client/src/components/More.jsx [hunk 1]
```diff
@@ -274,6 +283,7 @@ const More = ({ setSubscribedNumber, unsubscribeNumber, subscribedNumber, clearR
               className={classes.buttons}
               onClick={() => {
                 handleClose();
+                setReminderStatus(true);
```
Comment (AdhamAH) [inline client/src/components/More.jsx:286]: is the aim to show the dialogue again if the user sings out and then in? if so it didn't work

File: client/src/components/DiagnosticContainer.jsx [hunk 2]
```diff
@@ -51,10 +48,46 @@ function DiagnosticContainer() {
       </Typography>
       <hr className={classes.hr} />
       <HealthLogToggle />
+      <Dialog
```
Comment (AdhamAH) [inline client/src/components/DiagnosticContainer.jsx:51]: Could you make the background colour as the app's background colour?

Comment (AdhamAH) [reply client/src/components/DiagnosticContainer.jsx:51]: I looked a little bit farther into it. It does work but I needed to refresh. not sure yet why

**Comment by AdhamAH** [comment]
I looked a little bit farther into it. It does work but I needed to refresh. not sure yet why

File: client/src/components/More.jsx (hunk 1 above)
Comment (SomeMoosery) [reply client/src/components/More.jsx:286]: I think it's just if you haven't completed a survey on that calendar date, right?

Comment (AdhamAH) [reply client/src/components/More.jsx:286]: Oh yeah! if it is only shown again if you haven't completed a survey 'today' then it makes sense

**Comment by PhurbaSherpa** [comment]
> I looked a little bit farther into it. It does work but I needed to refresh. not sure yet why

I noticed I set surveyPage to zero in the clearSurvey action instead of setting it to surveyPage 1, just fixed it

**Comment by SomeMoosery** [comment]
Just need an approval from @AdhamAH as he requested changes and this should be good to go

**Pull request merged**